scripts/export-years

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
This was mainly due to a request from 360Giving about splitting a large dataset into yearly blocks. Including it here for completeness' sake.

Reply:
Any more context on what this does?

Reply:
Yeah, sorry – you call it like this:
`./scripts/export-years 2004 2019 grants-2019-01-22 ~/Desktop/years`

This will output a JSON file (`2004.json`, `2005.json`, etc) to the specified folder containing all grants for that year. 

Probably not useful right away as the data formats aren't interchangeable yet, but wanted to store it somewhere.

Reply:
👍